src/ui/material/form-field/src/form-field.wrapper.ts

```diff
@@ -8,21 +8,36 @@ import {
   AfterViewInit,
   TemplateRef,
   ElementRef,
-  ViewContainerRef,
 } from '@angular/core';
 import {
   ɵdefineHiddenProp as defineHiddenProp,
-  ɵobserve as observe,
   FormlyFieldConfig,
+  FormlyTemplateOptions,
   FieldWrapper,
 } from '@ngx-formly/core';
 import { MatFormField } from '@angular/material/form-field';
 import { FocusMonitor } from '@angular/cdk/a11y';
+import { FloatLabelType, MatFormFieldAppearance } from '@angular/material/form-field/form-field';
+import { ThemePalette } from '@angular/material/core';
 
-interface MatFormlyFieldConfig extends FormlyFieldConfig {
-  _matprefix: TemplateRef<any>;
-  _matsuffix: TemplateRef<any>;
+export interface MatFormlyFieldConfig extends FormlyFieldConfig {
+  _matprefix?: TemplateRef<any>;
+  _matsuffix?: TemplateRef<any>;
   _formField?: FormlyWrapperFormField;
+  templateOptions?: MatFormlyTemplateOptions;
+}
+
+export interface MatFormlyTemplateOptions extends FormlyTemplateOptions {
```

Comment:
no need to export `MatFormlyFieldConfig`, as its part of `MatFormlyFieldConfig` so it can be used through it: 

```patch
- MatFormlyFieldConfig
+ MatFormlyFieldConfig['templateOptions']
```